docs/tutorials/mini.md

```diff
@@ -14,112 +14,24 @@ installed
-                        MA_mock-A36971_batch-s4W2Go4tinn49nkhSuusrE-2 is NOT SUBMITTED
-                        MA_mock-A47933_batch-s4W2Go4tinn49nkhSuusrE-1 is NOT SUBMITTED
-                        MA_mock-A47933_batch-s4W2Go4tinn49nkhSuusrE-2 is NOT SUBMITTED
-                        MA_mock-A47933_batch-s4W2Go4tinn49nkhSuusrE-3 is NOT SUBMITTED
-[2018-06-01 12:19:31] pairing
-                        MP_batch-s4W2Go4tinn49nkhSuusrE is NOT SUBMITTED
-[2018-06-01 12:19:31] summary
-                        MS_batch-s4W2Go4tinn49nkhSuusrE is NOT SUBMITTED
-                        rewriting: output_dir/build.cfg
+mavis/schemas/config.json
 ```
 
-Adding the submit argument will start the pipeline
+Now you are ready to run MAVIS. This can be done in a single command using snakemake.
 
 ```bash
-mavis schedule -o output_dir/ --submit
-```
-
-After this completes, run schedule without the submit flag again and you
-should see something like
-
-```text
-                        MAVIS: 1.8.4
-                        hostname: gphost08.bcgsc.ca
-[2018-06-01 13:15:28] arguments
-                        command = 'schedule'
-                        log = None
-                        log_level = 'INFO'
-                        output = 'output_dir/'
-                        resubmit = False
-                        submit = False
-[2018-06-01 13:15:28] validate
-                        MV_mock-A36971_batch-s4W2Go4tinn49nkhSuusrE-1 (zQJYndSMimaoALwcSSiYwi) is COMPLETED
-                        MV_mock-A36971_batch-s4W2Go4tinn49nkhSuusrE-2 (BHFVf3BmXVrDUA5X4GGSki) is COMPLETED
-                        MV_mock-A47933_batch-s4W2Go4tinn49nkhSuusrE-1 (tUpx3iabCrpR9iKu9rJtES) is COMPLETED
-                        MV_mock-A47933_batch-s4W2Go4tinn49nkhSuusrE-2 (hgmH7nqPXZ49a8yTsxSUWZ) is COMPLETED
-                        MV_mock-A47933_batch-s4W2Go4tinn49nkhSuusrE-3 (cEoRN582An3eAGALaSKmpJ) is COMPLETED
-[2018-06-01 13:15:28] annotate
-                        MA_mock-A36971_batch-s4W2Go4tinn49nkhSuusrE-1 (tMHiVR8ueNokhBDnghXYo6) is COMPLETED
-                        MA_mock-A36971_batch-s4W2Go4tinn49nkhSuusrE-2 (AsNpNdvUyhNtKmRZqRSPpR) is COMPLETED
-                        MA_mock-A47933_batch-s4W2Go4tinn49nkhSuusrE-1 (k7qQiAzxfC2dnZwsGH7BzD) is COMPLETED
-                        MA_mock-A47933_batch-s4W2Go4tinn49nkhSuusrE-2 (dqAuhhcVKejDvHGBXn22xb) is COMPLETED
-                        MA_mock-A47933_batch-s4W2Go4tinn49nkhSuusrE-3 (eB69Ghed2xAdp2VRdaCJBf) is COMPLETED
-[2018-06-01 13:15:28] pairing
-                        MP_batch-s4W2Go4tinn49nkhSuusrE (6LfEgBtBsmGhQpLQp9rXmi) is COMPLETED
-[2018-06-01 13:15:28] summary
-                        MS_batch-s4W2Go4tinn49nkhSuusrE (HDJhXgKjRmseahcQ7mgNoD) is COMPLETED
-                        rewriting: output_dir/build.cfg
-                        run time (hh/mm/ss): 0:00:00
-                        run time (s): 0
+snakemake -j 1 --configfig tests/mini-tutorial.config.json
```

Comment:
typo in snakemake command

```
snakemake -j 1 --config tests/mini-tutorial.config.json
```

Reply:
good catch, will include in the docs update PR